src/Stubs.cs

```diff
@@ -31,4 +46,20 @@ public ReducerEventBase(TransactionUpdate update)
 
         public abstract bool InvokeHandler();
     }
+
+    public abstract class RemoteBase<DbConnection> {
+#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor.

```

Comment:
Why not just turn `Init` into an actual constructor?

Reply:
Ah hmm I'll recheck that one, I think I initially put that there because of the new() constraint